optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/PlannerBenchmarkResult.java

```diff
@@ -277,6 +283,7 @@ public void accumulateResults(BenchmarkReport benchmarkReport) {
 
     private void determineTotalsAndAverages() {
         failureCount = 0;
+        boolean environmentModeSet = false;
```

Comment:
Any variable ending with Set is usually expected to be a java.util.Set. For clarity, we might want to rename it.

Or just do a boolean first:

boolean first = true;
for (...) {
   if (first) {
      ...
      first = false;
   } else {
     ...
   }
   ...
}